tests/format/html/attributes/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -253,29 +253,17 @@ printWidth: 80
 
 =====================================output=====================================
 <div
-  class="
-    ProviderMeasuresContainer__heading-row
-    d-flex
-    flex-column flex-lg-row
-    justify-content-start justify-content-lg-between
-    align-items-start align-items-lg-center
-  "
```

Comment:
Can you add this output as a new test input to make sure that existing code with line breaks in the `class` attribute continues to be formatted correctly?

Reply:
This PR does not implement the behavior where it keeps it if there is already a line break. So they will get collapsed. 

I just realized why I never noticed this behavior before: In JSX, we just keep the class name as it exactly. It seems like the JSX and HTML behavior should be exactly the same. We should just keep the class name as is, so that means that all existing users won't be effected.

Reply:
This PR does not implement the behavior where it keeps it if there is already a line break. So they will get collapsed.

I just realized why I never noticed this behavior before: In JSX, we just keep the class name as it exactly. It seems like the JSX and HTML behavior should be exactly the same. We should just keep the class name as is, so that means that all existing users won't be effected.

Reply:
> In JSX, we just keep the class name as it exactly. It seems like the JSX and HTML behavior should be exactly the same. 

Formatting for class names in html was introduced in #7555  https://prettier.io/blog/2020/03/21/2.0.0.html#format-value-of-html-class-attribute-7555httpsgithubcomprettierprettierpull7555-by-fiskerhttpsgithubcomfisker